src/plugins/identity/alias.js

```diff
@@ -17,25 +17,33 @@ import {userAddress as discourseAddress} from "../discourse/address";
  */
 export type Alias = string;
 
+const _VALID_ALIAS = /^(\w+)\/@?(.*)$/;
```

Comment:
Although this will be covered by the regex for each username. Why not use `(.+)` for the username part?

Reply:
Type-specific parsers should handle that. Maybe we’ll add an `self/`
alias that always refers to the main project node or something, and has
no need for a username. The `.*` pattern is simpler than `.+`, so we
should prefer it unless there’s a reason not to.

Reply:
Yeah that was what I was thinking. To have a bit of bias over what the username specific form can look like (similar to stripping @'s). But so long as we're exposing the function and not the pattern, either works for me :]

Reply:
Yep, agreed. You’re right that this is a bit inconsistent; we should
probably push the `@`-stripping down, too. Done.